tests/system/Storage/RequesterPaysTest.php

```diff
@@ -116,10 +160,64 @@ public function testRequesterPaysMethodsWithoutUserProject(callable $call)
         $call($bucket, $object);
     }
 
+    /**
+     * @dataProvider requesterPaysMethods
+     */
+    public function testRequesterPaysWithUserProject(callable $call)
+    {
+        $bucket = $this->requesterPaysClient->bucket(self::$bucketName, $this->requesterProject);
+        $object = $bucket->object(self::$object1->name());
+
+        $call($bucket, $object);
+    }
```

Comment:
Can we make sure to have an assertion here? Otherwise this test would be marked as risky.